pkg/metrics/metrics.go

```diff
@@ -44,12 +46,18 @@ func NewMetrics() Metrics {
 			Help: "completed pipelines",
 		},
 			[]string{"result"}),
-		Duration: prometheus.NewHistogramVec(prometheus.HistogramOpts{
-			Name:    "steward_pipelinerun_duration_seconds",
-			Help:    "pipeline run durations",
+		TotalDuration: prometheus.NewHistogramVec(prometheus.HistogramOpts{
+			Name:    "steward_pipelinerun_total_duration_seconds",
+			Help:    "pipeline run durations after they changed their status",
 			Buckets: prometheus.ExponentialBuckets(0.125, 2, 15),
 		},
 			[]string{"state"}),
+		CurrentDuration: prometheus.NewHistogramVec(prometheus.HistogramOpts{
+			Name:    "steward_pipelinerun_current_duration_seconds",
+			Help:    "pipeline run durations in their current status",
+			Buckets: prometheus.ExponentialBuckets(60, 2, 7),
```

Comment:
Why do we have different bucket settings for ongoing and finished durations?

Reply:
For ongoing once it does not really make sense to have smaller buckets then the scrape and metering interval. The other buckets I simply did not change.

Reply:
I see. It fits to the purposes of the metric.